Source/WebCore/Modules/model-element/HTMLModelElement.h

```diff
@@ -222,6 +222,8 @@ class HTMLModelElement final : public HTMLElement, private CachedRawResourceClie
     void environmentMapRequestResource();
     void environmentMapResetAndReject(Exception&&);
     void environmentMapResourceFinished();
+    bool portal() const;
```

Comment:
Nit: I think to keep consistent with the method name in `ModelPlayer` (and also to make it clear that it's a boolean), this should be `hasPortal()`  and `updateHasPortal()`.